### PR TITLE
feat: [PROD-13016] implement logs download for failed runners runs

### PR DESCRIPTION
**Pull request by nborde-CSM**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: src/views/DatasetManager/components/DatasetOverview/components/DatasetOverviewPlaceholder/DatasetOverviewPlaceholder.js [hunk 1]
```diff
@@ -102,6 +104,18 @@ export const DatasetOverviewPlaceholder = () => {
       <ReuploadFileDatasetButton datasetId={currentDatasetId} iconButton={false} />
     ) : (
       <>
+        {!Object.values(DATASET_SOURCE_TYPE).includes(currentDatasetType) && (
+          <Button
+            sx={{ mr: 1 }}
+            data-cy={'successful-run-logs-download-button'}
```
Comment (csm-thu) [inline src/views/DatasetManager/components/DatasetOverview/components/DatasetOverviewPlaceholder/DatasetOverviewPlaceholder.js:110]: Can we change the data-cy here? This button is not used for "succesful runs" :slightly_smiling_face:

**Review by csm-thu — APPROVED** [review]
One suggestion, but LGTM :+1:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: cypress/commons/constants/generic/IdConstants.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ export const GENERIC_SELECTORS = {
       placeholder: '[data-cy=dashboard-placeholder]',
       accordion: '[data-cy=dashboards-accordion]',
       accordionSummary: '[data-cy=dashboards-accordion-summary]',
-      accordionLogsDownloadButton: '[data-cy=successful-run-logs-download-button]',
+      accordionLogsDownloadButton: '[data-cy=run-logs-download-button]',
```
Comment (csm-thu) [inline cypress/commons/constants/generic/IdConstants.js:67]: @nborde-CSM The previous value "_successful-run-logs-download-button_" for this selector is still used in _src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js_, we should probably keep the previous one for scenario download logs and create a new selector for the runner logs download

**Pull request merged**